src/components/EditableInput/lib/helper.ts

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
You import ReactElement here so you don't have to use React.ReactElement for the typing.
`import React, {ReactElement} from 'react'`

Reply:
```suggestion
import React, {ReactElement} from 'react';
```